_docs/identity.md

```diff
@@ -1,73 +1,86 @@
+| Out of Date      | The information used to be good, but is now out of date.                                      |
+| Low Quality      | The information is low quality or imprecise, but can be fixed with an update.                 |
+| Erroneous        | The information is erroneous and may indicate malicious intent.                               |
 
-![request-judgement](../../assets/img/identity/request-judgement.png)
+### Step 7: Check your identity
 
-- regIndex: The index of registrars.
-- maxFee: You need to provide a fee in `Planck` higher than registrar.
+After the identity is approved, you can see a green symbol on the left side of the your address, indicating that the identity verification is approved.
 
-## Asks registrar to approve judgement
+![success](../../assets/img/identity/success.png)
 
-### Contact Registrar
+## FAQ
 
-You can ask the registrar to review your request in [OAK Discord room](https://discord.gg/7W9UDvsbwh).
+### Why do I need to set an identity on-chain?
 
-### Registrar approves judgement
+There are a number of reasons to set an on-chain identity or remain anonymous. For the following users and token holders, we recommend setting an on-chain identity to make participation more seamless and efficient.
 
-When a registrar provides judgement, they can select up to six levels of confidence in their attestation:
+- Participants of governance (council members, committee members, motion passers, etc.)
+- Collators seeking delegators to stake on their behalf
+- Community members seeking to be registrars
 
-- Unknown: The default value, no judgement made yet.
+### Who approves someone's identity?
 
-- Reasonable: The data appears reasonable, but no in-depth checks (e.g. formal KYC process) were performed.
+Registrars. At first, registrars are set via Sudo, and are OAK team members or ambassadors. Once governance is enabled, any community member can become registrars via democracy.
 
-- Known Good: The registrar has certified that the information is correct.
+There is no KYC/AML process to set an on-chain identity. However, if there is very clearly spam or misuse, it might be difficult for a registrar to understand intent behind setting on-chain identity.
 
-- Out of Date: The information used to be good, but is now out of date.
+### Which registrar should I ask for a judgement?
 
-- Low Quality: The information is low quality or imprecise, but can be fixed with an update.
+You will have to select which registrar to pick by vetting their identity. To do so, do some research on their account. You can start by navigating to [Turing's Subscan Accounts](https://turing.subscan.io/account) page and searching for their account.
 
-- Erroneous: The information is erroneous and may indicate malicious intent.
+![registrars](../../assets/img/identity/registrars.png)
 
-## Check your indentity
+As shown above, there are currently two registrars.
```

Comment:
These screenshots and info are from turing staging. Might be confusing when folks are expecting turing proper. Especially since there are specific wallet addresses in this guide.

Not a high priority to fix, but maybe something to consider updating in the future.